src/components/ActionBar/src/AtomicActionBar.vue

```diff
@@ -36,17 +38,28 @@ export default {
 			validator: (hideOn) => ['none', 'mobile', 'tablet', 'desktop'].includes(hideOn),
 		},
 	},
+
+	computed: {
+		hasSafariAdjustment() {
+			return isMobileSafari();
+		},
+	},
 };
 </script>
 
 <style module="$s">
 .ActionBar {
-	--action-bar-bottom-padding: 64px;
+	--action-bar-padding: 24px;
```

Comment:
Shouldn't this be 32px? [The line you deleted has it as 32px](https://github.com/square/maker/pull/80/files#diff-6137173e29b51786287282c92fedffb9a92f90d5605a9203d1079fe83711ddf2L64).

Reply:
Design would prefer to change it to 24px.